server/embed/config.go

```diff
@@ -644,7 +644,7 @@ func (cfg *Config) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&cfg.InitialClusterToken, "initial-cluster-token", cfg.InitialClusterToken, "Initial cluster token for the etcd cluster during bootstrap.")
 	fs.BoolVar(&cfg.StrictReconfigCheck, "strict-reconfig-check", cfg.StrictReconfigCheck, "Reject reconfiguration requests that would cause quorum loss.")
 
-	fs.BoolVar(&cfg.PreVote, "pre-vote", cfg.PreVote, "Enable to run an additional Raft election phase.")
+	fs.BoolVar(&cfg.PreVote, "pre-vote", cfg.PreVote, "Enable a Raft voting phase before an election in which followers will not grant their vote if they believe the leader is alive & in which the candidate's term will not increment.")
```

Comment:
Accurate, but not very user friendly. I think it would be good to mention why someone should enable it.

Something like:

Prevents a reconnecting member forcing a needless leader election on healthy cluster, disrupting cluster availability.